### ISSUE TITLE
Incorrect transform in preprocessing.py

### ISSUE THREAD

**Issue by MatthewKleinsmith** [PetroWu/AutoPortraitMatting#31]
## With transform in preprocessing.py

![bad2](https://user-images.githubusercontent.com/8968171/32228318-2b8314b2-be0b-11e7-8f3a-1dbef0afb3f4.png)

## With correct transform
![good](https://user-images.githubusercontent.com/8968171/32228131-aa62050a-be0a-11e7-84a5-7e152f9fc49d.png)

## Fix

https://github.com/PetroWu/AutoPortraitMatting/pull/32

## Discussion

The bottom center image of each example is the mean mask aligned with the given portrait. They're intended to be rough estimates of what the final mask should look like. The misalignment in the first example is due to using an incorrect transform.

**Comment by MLjian** [comment]
Hi，I am doing my homework, I have some questions about it. Hope to communicate.

**Comment by MatthewKleinsmith** [comment]
@MLjian What's your question?

**Comment by MLjian** [comment]
Did you run the example? How is the result?

**Comment by MatthewKleinsmith** [comment]
I'm stuck at 90% IoU on a validation set, while the authors got 96% on the test set. This is with my custom code, though.

I haven't ran this repo's code. I only looked at the code to compare my implementation of the paper with PetruWu's implementation. I noticed an error, and so I posted here.

**Comment by MLjian** [comment]
You are so passionate. Thank you very much.I saw the paper Automatic  Portrait Segmentation for Image Stylization carefully. And what's the architecture in deep learning for implement? Tensorflow or other.

**Comment by MatthewKleinsmith** [comment]
@MLjian 

The paper uses a custom Caffe build from 2014. I'm using PyTorch.

**Comment by MLjian** [comment]
Hi,guy!
The datasets folder are so complex for me. Now I want to just use the FCN without additinal channels to complete the matting. So I want to know whether the Images_mask folder is the annotation that the paper author labeled.
